x-pack/plugins/ml/public/data_frame/pages/data_frame_new_pivot/wizard.tsx

```diff
@@ -0,0 +1,181 @@
+  isCurrentStep,
+  pivotState,
+  setCurrentStep,
+  setPivot,
+}) => {
+  const definePivotRef = useRef(null);
+
+  return (
+    <Fragment>
+      <div ref={definePivotRef} />
+      {isCurrentStep && (
+        <Fragment>
+          <DefinePivotForm onChange={setPivot} overrides={pivotState} />
+          <WizardNav
+            next={() => setCurrentStep(WIZARD_STEPS.JOB_DETAILS)}
+            nextActive={pivotState.valid}
+          />
+        </Fragment>
+      )}
+      {!isCurrentStep && <DefinePivotSummary {...pivotState} />}
+    </Fragment>
+  );
+};
+
+export const Wizard: SFC = React.memo(() => {
+  // indexPattern from context
+  const indexPattern = useContext(IndexPatternContext);
+
+  if (indexPattern === null) {
+    return null;
+  }
+
+  // The current WIZARD_STEP
+  const [currentStep, setCurrentStep] = useState(WIZARD_STEPS.DEFINE_PIVOT);
+
+  // The DEFINE_PIVOT state
+  const [pivotState, setPivot] = useState(getDefaultPivotState());
+
+  // The JOB_DETAILS state
+  const [jobDetailsState, setJobDetails] = useState(getDefaultJobDetailsState());
+
+  const jobDetails =
+    currentStep === WIZARD_STEPS.JOB_DETAILS ? (
+      <JobDetailsForm onChange={setJobDetails} overrides={jobDetailsState} />
+    ) : (
+      <JobDetailsSummary {...jobDetailsState} />
+    );
+
+  // The JOB_CREATE state
+  const [jobCreateState, setJobCreate] = useState(getDefaultJobCreateState);
+
+  const jobCreate =
+    currentStep === WIZARD_STEPS.JOB_CREATE ? (
+      <JobCreateForm
+        jobId={jobDetailsState.jobId}
+        jobConfig={getDataFrameRequest(indexPattern.title, pivotState, jobDetailsState)}
+        onChange={setJobCreate}
+        overrides={jobCreateState}
+      />
+    ) : (
+      <JobCreateSummary />
+    );
+
+  // scroll to the currently selected wizard step
+  /*
+  function scrollToRef() {
+    if (definePivotRef !== null && definePivotRef.current !== null) {
+      // TODO Fix types
+      const dummy = definePivotRef as any;
+      const headerOffset = 70;
+      window.scrollTo(0, dummy.current.offsetTop - headerOffset);
+    }
+  }
+  */
+
+  const stepsConfig = [
+    {
+      title: i18n.translate('xpack.ml.dataframe.transformsWizard.definePivotStepTitle', {
+        defaultMessage: 'Define pivot',
+      }),
+      children: (
+        <DefinePivotStep
+          isCurrentStep={currentStep === WIZARD_STEPS.DEFINE_PIVOT}
+          pivotState={pivotState}
+          setCurrentStep={setCurrentStep}
+          setPivot={setPivot}
+        />
+      ),
+    },
+    {
+      title: i18n.translate('xpack.ml.dataframe.transformsWizard.jobDetailsStepTitle', {
+        defaultMessage: 'Job details',
+      }),
+      children: (
+        <Fragment>
+          {jobDetails}
+          {currentStep === WIZARD_STEPS.JOB_DETAILS && (
+            <WizardNav
+              previous={() => {
+                setCurrentStep(WIZARD_STEPS.DEFINE_PIVOT);
+                // scrollToRef();
```

Comment:
Is this scrolling a WIP?

Reply:
I want to re-enable this but it's broken for now because of a previous refactor. I added a note to the Github project to address it later.